packages/data-model/src/functions/migrate.ts

```diff
@@ -0,0 +1,36 @@
+import 'dotenv/config';
+
+import { drizzle } from 'drizzle-orm/node-postgres';
+import { migrate as migrator } from 'drizzle-orm/node-postgres/migrator';
+import path from 'path';
+import pg from 'pg';
+import { fileURLToPath } from 'url';
+
+const getRequiredConfig = (name: string) => {
+	const value = process.env[name];
+	if (!value) {
+		throw new Error(`No Environment Variable provided for required configuration parameter '${name}'`);
+	}
+	return value;
+};
+
+export async function migrate() {
+	console.log('Running your migrations...');
+
+	const sql = new pg.Pool({
+		host: getRequiredConfig('DB_HOST'),
+		database: getRequiredConfig('DB_NAME'),
+		password: getRequiredConfig('DB_PASSWORD'),
+		port: Number(getRequiredConfig('DB_PORT')),
+		user: getRequiredConfig('DB_USER'),
+	});
```

Comment:
Are we not exporting this in a way that uses dependencies like our config? When imported this puts requirements on the environment rather than just being a code library.

Reply:
You're right!. thanks for pointing this out. As this package `data-model` is imported in the `lyric-provider`, I will create here the `DbConfig` definition type to make it consistent along both packages.